### ISSUE TITLE
Theme is broken

### ISSUE THREAD

**Issue by daninus14** [onesine/react-tailwindcss-datepicker#175]
As it stands the theme is chosen based on the system choice, and not on the tailwind choice. Even if the tailwind config is not setting the dark mode, the component will be displayed in dark mode.

There should be a prop to choose the theme, and then another way to base it on the tailwind config and not on the user's system alone...

**Comment by daninus14** [comment]
After restarting the server, the light theme loaded, but it still should be easier to change the theme by a prop.

Thanks!

**Comment by murilosr** [comment]
Yes, I had the same problem. I didn't set a color theme, but it was using my system's one (dark), but the rest of the page was on the default (white mode), so it was weird.

This was how I fixed it:

``` javascript

// Insert this on tailwind.config.js
{
    ...
    darkMode: 'class',
    ...
}
```

Useful readings:

- [Toggling dark mode manually](https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually)
- [Supporting system preference and manual selection](https://tailwindcss.com/docs/dark-mode#supporting-system-preference-and-manual-selection)

**Comment by daninus14** [comment]
@murilosr thank you so much for the workaround and for the tailwind doc links!

**Comment by HurSungYun** [comment]
Same here. Fixed it using the workaround @murilosr suggested. Thank you.
